### DISCUSSION TITLE
[Question] Comparison of use cases of SCH x SCH_LAB

### DISCUSSION THREAD

**Discussion by igorluppi** [nasa/cFS#880] [General]
**Usability and use-case questions:**
- As far as I understood, the open issue https://github.com/nasa/SCH/issues/25 has no target date to be reached. How close is SCH to being CFE draco-rc4 compliant?
- Could you explain the differences in approach between SCH and SCH_LAB?
- Can the features of SCH_LAB fully comply with the capabilities of SCH 2.2.2?
- Is SCH_LAB a test or mockup app? I mean, is it not supposed to be production compliant and only used for simulation purposes?
- Is the SCH app required for draco-rc4 operation? I mean, is SCH_LAB the only one supposed to be working with draco-rc4? 

**Describe the solution you'd like:**
I would like a clear and objective description of the use-cases and usability of both apps, including guidance on when to use one over the other or if both are needed.

**Comment by dzbaker** [comment]
Moving to discussions (on main cFS repository).
